src/ui/component/filePicker/filePicker.tsx

```diff
@@ -22,6 +21,38 @@ type FilePickerProps = {
   onError?: (error: Error) => void
 }
 
+const extractFileStream = (file: globalThis.File): File =>
+  Object.assign(file, {
+    stream: file.stream()
+  })
+
+const mapDataTransferItemToFiles = (item: DataTransferItem): File[] => {
+  const file = item.getAsFile()
+  if (!file) throw new Error('Impossible to get file.')
+  return [extractFileStream(file)]
+}
+
+const traverseDirectory = async (fsDirectory: FileSystemDirectoryEntry): Promise<File[]> => {
+  const files: File[] = []
+  const directoryReader = fsDirectory.createReader()
+
+  const entries = await new Promise<FileSystemEntry[]>(resolve =>
```

Comment:
I think TS infers this:
```suggestion
  const entries = await new Promise(resolve =>
```

Reply:
Sadly now, it appears to be `unknown` without it :(